condax/__init__.py

```diff
@@ -1 +1,8 @@
-__version__ = "0.0.5"
```

Comment:
Alternatively we could keep this as the single-source-of-truth for the version number. I personally prefer this, and then I have a CD workflow which creates Git tags. (I used `source = "vcs"` on conda-lock because it was using `setuptools-scm`, which behaves similarly.) Does @mariusvniekerk have an opinion here?

Reply:
I am of the opinion that the version should not be present in the code. I typically set up my projects' CI for the deployment to happen when I create a GH release (which includes the creation of a tag). So it goes something like this:
`Create git tag > triggers deploy pipeline > builds project (taking version from new tag) and uploads it to pypi/conda`

Reply:
Fine by me, I was just curious if @mariusvniekerk has a preference, but since he hasn't said anything I'd assume not.